docs/admin/cluster-management.md

```diff
@@ -22,41 +22,33 @@ certainly want the docs that go with that version.</h1>
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Cluster Management
 
-This doc is in progress.
-
 ## Upgrading a cluster
 
-The `cluster/kube-push.sh` script will do a rudimentary update; it is a 1.0 roadmap item to have a robust live cluster update system.
-
-## Updgrading to a different API version
-
-There is a sequence of steps to upgrade to a new API version.
-
-1. Turn on the new api version
-2. Upgrade the cluster's storage to use the new version.
-3. Upgrade all config files. Identify users of the old api version endpoints.
-4. Update existing objects in the storage to new version by running cluster/update-storage-objects.sh
-3. Turn off the old version.
-
-### Turn on or off an API version for your cluster
-
-Specific API versions can be turned on or off by passing --runtime-config=api/<version> flag while bringing up the server. For example: to turn off v1 API, pass --runtime-config=api/v1=false.
-runtime-config also supports 2 special keys: api/all and api/legacy to control all and legacy APIs respectively. For example, for turning off all api versions except v1, pass --runtime-config=api/all=false,api/v1=true.
-
-### Switching your cluster's storage API version
+The current state of cluster upgrades is provider dependent.
 
-KUBE_API_VERSIONS env var controls the API versions that are supported in the cluster. The first version in the list is used as the cluster's storage version. Hence, to set a specific version as the storage version, bring it to the front of list of versions in the value of KUBE_API_VERSIONS.
+### Upgrading Google Container Engine (GKE)
+In Google Container Engine, your apiserver will always be automatically updated to the latest version.
+Updating your nodes is described in the [GKE documentation](https://cloud.google.com/container-engine/docs/clusters/upgrade)
 
-### Switching your config files to a new API version
-
-You can use the kube-version-change utility to convert config files between different API versions.
+### Upgrading open source Google Compute Engine clusters
+Upgrades on open source Google Compute Engine (GCE) clusters are controlled by the ```cluster/gce/upgrade.sh``` script.
 
+It's usage is as follows:
 ```
-$ hack/build-go.sh cmd/kube-version-change
-$ _output/local/go/bin/kube-version-change -i myPod.v1beta3.yaml -o myPod.v1.yaml
+./cluster/gce/upgrade.sh [-M|-N|-P] -l | <release or continuous integration version> | [latest_stable|latest_release|latest_ci]
+  Upgrades master and nodes by default
+  -M:  Upgrade master only
+  -N:  Upgrade nodes only
+  -P:  Node upgrade prerequisites only (create a new instance template)
+  -l:  Use local(dev) binaries
 ```
 
-### Maintenance on a Node
+### Other platforms
+
+The `cluster/kube-push.sh` script will do a rudimentary update.  This process is still quite experimental, we
+recommend testing the upgrade on an experimental cluster before performing the update on a production cluster.
+
+## Maintenance on a Node
```

Comment:
I can't comment on lines you didn't touch, for the enumeration below "after "any of the following strategies:", could you please:
- make the list bullets instead of enumerated (otherwise they look like sequential steps, when really they are options)
- make the `kubectl` commands in their own indented block rather than inline
- capitalize all of the them and end them with a period
- add a newline before "If you deleted the node's...", keeping it indented

?